index.js

```diff
@@ -44,16 +46,22 @@ module.exports = {
           replacement: "import lodash from 'lodash'\nconst isPlainObject = lodash.isPlainObject"
         }
       ]
-    })
+    });
+
+    reduxTree = esTranspiler(reduxTree, {
```

Comment:
No need to do this manually via `broccoli-babel-transpiler` here.  You should use `ember-cli-babel`'s new functionality (documented [here](https://github.com/babel/ember-cli-babel#addon-usage)):

```js
  let babelAddon = this.addons.find(addon => addon.name === 'ember-cli-babel'); // find your babel addon

  let babelOptions = addon.buildBabelOptions({
    babel: {
      plugins: [
        [require('babel-plugin-transform-object-rest-spread').default]
      ]
    }
  });

  let reduxTree = babel.transpileTree(reduxTree, babelOptions);
```